crypto/mem.c

```diff
@@ -321,3 +278,53 @@ void CRYPTO_clear_free(void *str, size_t num, const char *file, int line)
         OPENSSL_cleanse(str, num);
     CRYPTO_free(str, file, line);
 }
+
+#if !defined(OPENSSL_NO_CRYPTO_MDEBUG)
+
+# ifndef OPENSSL_NO_DEPRECATED_3_0
+int CRYPTO_mem_ctrl(int mode)
+{
+    (void)mode;
+    return -1;
+}
+
+int CRYPTO_set_mem_debug(int flag)
+{
+    return -1;
```

Comment:
`(void)flag;` to stay consistent with the other functions here?
Or remove the casts in the others.

Reply:
I added the cast.  I don't know if some "unused" warning complains, but happy to remove all of them if that's preferred (in a separate PR).